src/com/epam/task/string/Task10.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.task.string;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static com.epam.task.util.InputUtils.inputString;
+
+public class Task10 {
+    private static final String REGEX = "[.?!]";
+
+    public static void main(String[] args) {
+        String string = inputString("Enter text : ");
+        int countOfSentencesInText = countSentence(string);
+        System.out.println(countOfSentencesInText);
+    }
+
+    public static int countSentence(String string) {
```

Comment:
It will be much easier to split string by your regex and count the length of the result array-it'll be number of the sentences